File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/LoadGrdBAS0260U01Info.java
package nta.med.data.model.ihis.bass;

import java.math.BigDecimal;
import java.math.BigInteger;

public class LoadGrdBAS0260U01Info {
	
	private BigInteger id;
	private String buseoCode;
	private String buseoName;
	private String buseoName2;
	private String buseoGubun;
	private String parentBuseo;
	private String gwa;
	private String gwaName;
	private String gwaName2;
	private String gwaGubun;
	private String parentGwa;
	private String note;
	private String language;
	private BigDecimal activeFlg;
	
	public LoadGrdBAS0260U01Info(BigInteger id, String buseoCode, String buseoName, String buseoName2,
			String buseoGubun, String parentBuseo, String gwa, String gwaName, String gwaName2, String gwaGubun,
			String parentGwa, String note, String language, BigDecimal activeFlg) {
		super();
		this.id = id;
		this.buseoCode = buseoCode;
		this.buseoName = buseoName;
		this.buseoName2 = buseoName2;
		this.buseoGubun = buseoGubun;
		this.parentBuseo = parentBuseo;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.gwaName2 = gwaName2;
		this.gwaGubun = gwaGubun;
		this.parentGwa = parentGwa;
		this.note = note;
		this.language = language;
		this.activeFlg = activeFlg;
	}

	public BigInteger getId() {
		return id;
	}

	public void setId(BigInteger id) {
		this.id = id;
	}

	public String getBuseoCode() {
		return buseoCode;
	}

	public void setBuseoCode(String buseoCode) {
		this.buseoCode = buseoCode;
	}

	public String getBuseoName() {
		return buseoName;
	}

	public void setBuseoName(String buseoName) {
		this.buseoName = buseoName;
	}

	public String getBuseoName2() {
		return buseoName2;
	}

	public void setBuseoName2(String buseoName2) {
		this.buseoName2 = buseoName2;
	}

	public String getBuseoGubun() {
		return buseoGubun;
	}

	public void setBuseoGubun(String buseoGubun) {
		this.buseoGubun = buseoGubun;
	}

	public String getParentBuseo() {
		return parentBuseo;
	}

	public void setParentBuseo(String parentBuseo) {
		this.parentBuseo = parentBuseo;
	}

	public String getGwa() {
		return gwa;
	}

	public void setGwa(String gwa) {
		this.gwa = gwa;
	}

	public String getGwaName() {
		return gwaName;
	}

	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}

	public String getGwaName2() {
		return gwaName2;
	}

	public void setGwaName2(String gwaName2) {
		this.gwaName2 = gwaName2;
	}

	public String getGwaGubun() {
		return gwaGubun;
	}

	public void setGwaGubun(String gwaGubun) {
		this.gwaGubun = gwaGubun;
	}

	public String getParentGwa() {
		return parentGwa;
	}

	public void setParentGwa(String parentGwa) {
		this.parentGwa = parentGwa;
	}

	public String getNote() {
		return note;
	}

	public void setNote(String note) {
		this.note = note;
	}

	public String getLanguage() {
		return language;
	}

	public void setLanguage(String language) {
		this.language = language;
	}

	public BigDecimal getActiveFlg() {
		return activeFlg;
	}

	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}
	
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/LoadGrdBAS0260U01Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    public class LoadGrdBAS0260U01Info
    {
        private String _id;
        private String _buseoCode;
        private String _buseoName;
        private String _buseoName2;
        private String _buseoGubun;
        private String _parentBuseo;
        private String _gwa;
        private String _gwaName;
        private String _gwaName2;
        private String _gwaGubun;
        private String _parentGwa;
        private String _note;
        private String _language;
        private String _activeFlg;
        private String _rowState;

        public String Id
        {
            get { return this._id; }
            set { this._id = value; }
        }

        public String BuseoCode
        {
            get { return this._buseoCode; }
            set { this._buseoCode = value; }
        }

        public String BuseoName
        {
            get { return this._buseoName; }
            set { this._buseoName = value; }
        }

        public String BuseoName2
        {
            get { return this._buseoName2; }
            set { this._buseoName2 = value; }
        }

        public String BuseoGubun
        {
            get { return this._buseoGubun; }
            set { this._buseoGubun = value; }
        }

        public String ParentBuseo
        {
            get { return this._parentBuseo; }
            set { this._parentBuseo = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String GwaName2
        {
            get { return this._gwaName2; }
            set { this._gwaName2 = value; }
        }

        public String GwaGubun
        {
            get { return this._gwaGubun; }
            set { this._gwaGubun = value; }
        }

        public String ParentGwa
        {
            get { return this._parentGwa; }
            set { this._parentGwa = value; }
        }

        public String Note
        {
            get { return this._note; }
            set { this._note = value; }
        }

        public String Language
        {
            get { return this._language; }
            set { this._language = value; }
        }

        public String ActiveFlg
        {
            get { return this._activeFlg; }
            set { this._activeFlg = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public LoadGrdBAS0260U01Info() { }

        public LoadGrdBAS0260U01Info(String id, String buseoCode, String buseoName, String buseoName2, String buseoGubun, String parentBuseo, String gwa, String gwaName, String gwaName2, String gwaGubun, String parentGwa, String note, String language, String activeFlg, String rowState)
        {
            this._id = id;
            this._buseoCode = buseoCode;
            this._buseoName = buseoName;
            this._buseoName2 = buseoName2;
            this._buseoGubun = buseoGubun;
            this._parentBuseo = parentBuseo;
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._gwaName2 = gwaName2;
            this._gwaGubun = gwaGubun;
            this._parentGwa = parentGwa;
            this._note = note;
            this._language = language;
            this._activeFlg = activeFlg;
            this._rowState = rowState;
        }

    }
}